### ISSUE TITLE
feature: a show source button or similar

### ISSUE THREAD

**Issue by pietroppeter** [pietroppeter/nimibook#60]
- not a feature that is present in original mdbook but it is something which makes a lot of sense: the idea is to have a way to directly show the source code of the page as we do in nimib
- since this is not native in mdbook what needs to be decided is how this elements should appear in the interface: a button? where?

**Comment by HugoGranstrom** [comment]
Agreed, always nice to show the simple source code behind an impressive page :) Here are my two cents: 

Both on mobile and desktop we have some space at the bottom of the screen. So we could place it like we do in ordinary nimib. Maybe we would want to center it here though. 
The other option I can come up with is adding a button to the header, but it gets a bit too crammed on mobile if you have a longer book title.

**Comment by pietroppeter** [comment]
On mobile it could be here. Definitely header is already packed

![image](https://user-images.githubusercontent.com/4997234/214659852-c8268ff6-39ec-4804-a868-5f01f4e03d04.jpeg)

**Comment by HugoGranstrom** [comment]
Yes, that should work :+1: I looked at the HTML template and we will have to add the desktop button separately as the chapter buttons along with their parent div is hidden on desktop.

**Comment by HugoGranstrom** [comment]
I have given this a small try, but the CSS-part of my brain isn't collaborating... So I'll skip this in the current PR, and we'll see if I revisit this later (or if someone else does it before me).
